x/globalfee/types/params_test.go

```diff
@@ -71,3 +73,84 @@ func Test_validateParams(t *testing.T) {
 		})
 	}
 }
+
+func Test_validateBypassMinFeeMsgTypes(t *testing.T) {
+	tests := map[string]struct {
+		msgTypes  interface{}
+		expectErr bool
+	}{
+		"DefaultParams, pass": {
+			DefaultParams().BypassMinFeeMsgTypes,
+			false,
+		},
+		"wrong msg type should make conversion fail, fail": {
+			[]int{0, 1, 2, 3},
+			true,
+		},
+		"empty msg types, pass": {
+			[]string{},
+			false,
+		},
+		"empty msg type, fail": {
+			[]string{""},
+			true,
+		},
+		"invalid msg type name, fail": {
+			[]string{"ibc.core.channel.v1.MsgRecvPacket"},
+			true,
+		},
+		"mixed valid and invalid msgs, fail": {
+			[]string{
+				"/ibc.core.channel.v1.MsgRecvPacket",
+				"",
```

Comment:
Should just use the invalid example from above

Reply:
They aren't the same. A leading "/" is missing in the msg-type above.